sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs

```diff
@@ -1286,9 +1251,8 @@ public new AppendBlobClient WithEncryptionScope(string encryptionScope)
                         response = await AppendBlobRestClient.AppendBlockAsync(
                             contentLength: (content.Length - content.Position),
                             body: content,
-                            // TODO #27253
-                            //transactionalContentCrc64: hashResult?.StorageCrc64,
-                            transactionalContentMD5: transactionalContentMD5, //hashResult?.MD5,
+                            transactionalContentCrc64: hashResult?.StorageCrc64,
+                            transactionalContentMD5: hashResult?.MD5,
```

Comment:
Any issue if both `transactionalContentCrc64` and `transactionalContentMD5` are set?

Reply:
Service will reject the request. However, a caller can only set one, and the code that calculates this object will respect that.